### PR TITLE
Static discovery [7774]

### PR DESCRIPTION
**Pull request by EduPonz**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/168/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/169/)

File: docs/advanced.rst [hunk 1]
```diff
@@ -939,6 +939,237 @@ Simple EDP Attributes
...
+|                        | participant.                      |                   |               |
+|                        | see :ref:`staticLocators`.        |                   |               |
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<multicastLocator>`` | Multicast locator of the          |                   |               |
+|                        | participant.                      |                   |               |
+|                        | see :ref:`staticLocators`.        |                   |               |
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<reliabilityQos>``   | See the :ref:`reliability`        | |besteffort|      | |besteffort|  |
+|                        | section.                          +-------------------+               |
+|                        | section.                          | |reliable|        |               |
```
Comment (rsanchez15) [inline docs/advanced.rst]: Repeated word

File: docs/advanced.rst [hunk 2]
```diff
@@ -939,6 +939,237 @@ Simple EDP Attributes
...
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<durabilityQos>``    | See the                           | |volatile|        | |volatile|    |
+|                        | :ref:`SettingDataDurability`      +-------------------+               |
+|                        | :ref:`SettingDataDurability`      | |transientlocal|  |               |
+|                        | section.                          +-------------------+               |
```
Comment (rsanchez15) [inline docs/advanced.rst]: Repeated word

File: docs/advanced.rst [hunk 3]
```diff
@@ -939,6 +939,237 @@ Simple EDP Attributes
...
+|                        |                                   +-------------------+               |
+|                        |                                   | :class:`WITH_KEY` |               |
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<partitionQos>``     | The name of a partition of the    | ``string``        |               |
+|                        | remote peer. Repeat to configure  |                   |               |
+|                        | several partitions.               |                   |               |
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<unicastLocator>``   | Unicast locator of the            |                   |               |
+|                        | participant.                      |                   |               |
+|                        | see :ref:`staticLocators`.        |                   |               |
```
Comment (rsanchez15) [inline docs/advanced.rst]: Capital letter for "See..."

File: docs/advanced.rst [hunk 4]
```diff
@@ -939,6 +939,237 @@ Simple EDP Attributes
...
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<multicastLocator>`` | Multicast locator of the          |                   |               |
+|                        | participant.                      |                   |               |
+|                        | see :ref:`staticLocators`.        |                   |               |
```
Comment (rsanchez15) [inline docs/advanced.rst]: capital S

File: docs/advanced.rst [hunk 5]
```diff
@@ -939,6 +939,237 @@ Simple EDP Attributes
...
+|                        | section.                          +-------------------+               |
+|                        | section.                          | |transient|       |               |
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<ownershipQos>``     | See                               |                   |               |
+|                        | :ref:`ownershipQos`.              |                   |               |
++------------------------+-----------------------------------+-------------------+---------------+
+| ``<livelinessQos>``    | Defines the liveliness of the     |                   |               |
+|                        | remote peer.                      |                   |               |
+|                        | see :ref:`livelinessQos`.         |                   |               |
++------------------------+-----------------------------------+-------------------+---------------+
```
Comment (rsanchez15) [inline docs/advanced.rst]: Capital S

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/171/)

**Pull request merged**